examples/pytorch/ggnn/README.md

```diff
@@ -0,0 +1,62 @@
+# Gated Graph Neural Network (GGNN)
+
+- Paper link: https://arxiv.org/pdf/1511.05493.pdf
+
+## Dependencies
+- PyTorch 1.0+
+
+- DGL 0.3.1+
+
+## Solving bAbI tasks
+
```

Comment:
Could you please elaborate more about the dataset and task? This does not need to be perfect, but should provide an appropriate context for understanding the scripts and configurations you use below, including:
- What do "ns", "gc", "path_finding" stand for and what will be their semantic meanings in the particular scenario?
- What do settings like `task_id`, `question_id`, `question_type`, `train_num` refer to?